sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -620,29 +620,25 @@ AZ_NODISCARD az_result az_iot_pnp_client_twin_property_end_component(
  * **Without Component**
  * @code
  * {
- *   "reported":{
- *     "<property_name>":{
- *       "value": 23,
- *       "ac": <ack_code>,
- *       "av": <ack_version>,
- *       "ad": "<ack_description>"
- *     }
+ *   "<property_name>":{
+ *     "value": 23,
+ *     "ac": <ack_code>,
+ *     "av": <ack_version>,
+ *     "ad": "<ack_description>"
  *   }
  * }
  * @endcode
  *
  * **With Component**
  * @code
  * {
- *   "reported": {
```

Comment:
No more "reported" property in which components were nested? Just a typo or did you find something in testing?

Reply:
no this is correct. I had used JSON from an online doc as a reference and thought about it and it was wrong. I have a PR here which updates it:
https://github.com/MicrosoftDocs/azure-docs/issues/63135